pkg/machinery/compatibility/talos13/talos13.go

```diff
@@ -0,0 +1,26 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+// Package talos13 provides compatibility constants for Talos 1.3.
+package talos13
+
+import "github.com/hashicorp/go-version"
+
+// MajorMinor is the major.minor version of Talos 1.3.
+var MajorMinor = [2]int{1, 3}
+
+// MinimumHostUpgradeVersion is the minimum version of Talos that can be upgraded to 1.3.
+var MinimumHostUpgradeVersion = version.Must(version.NewVersion("1.0.0"))
+
+// MaximumHostDowngradeVersion is the maximum (not inclusive) version of Talos that can be downgraded to 1.3.
+var MaximumHostDowngradeVersion = version.Must(version.NewVersion("1.5.0"))
+
+// BlacklistedHostUpgradeVersions are the versions of Talos that cannot be upgraded to 1.3.
+var BlacklistedHostUpgradeVersions = []*version.Version{}
```

Comment:
```suggestion
var DenylistedHostUpgradeVersions = []*version.Version{}
```

Reply:
probably `DeniedHostUpgradeVersions` make sense

Reply:
yep, this looks better, thanks, I keep forgetting about it!